assets/tasks/common.json

```diff
@@ -1330,6 +1288,7 @@
         },
         {
             "label": "push-release-arm",
+            "detail": "These tasks will build the application based in the production Dockerfile and push the image to the DockerHub.",
```

Comment:
This task will build the application based on the production Dockerfile and push the image to the DockerHub.

Reply:
check last commit